### PR TITLE
Add linter configuration

### PR DESCRIPTION
**Pull request by thrunabilax**
Three commits:

1. Added plyintrc for pylint configuration
    
    Added configuration file pylintrc for increased stability when coding.
    This will enable code consistency and minimize coding mistakes.

2. Renamed pylintrc to .pylintrc
    
    Renamed pylintrc to .pylintrc for better consistency in naming config files

3. Renamed pylintrc to .pylintrc
    
    Renamed pylintrc to .pylintrc for better consistency in naming config files

### CODE REVIEW

File: .flake8 [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[flake8]
+exclude = .git
```
Comment (MasterMind2k) [inline .flake8]: Great!

Also please add this line, `import-order-style=pycharm`, this will set ordering of the modules.

File: .pylintrc [hunk 2]
```diff
@@ -0,0 +1,378 @@
+[MASTER]
```
Comment (MasterMind2k) [inline .pylintrc]: You used a default configuration file?

I think this is a good start. Let us continue with it.

File: .flake8 (hunk 1 above)
Comment (thrunabilax) [reply .flake8]: Line added as in flask-phbb3.

**Comment by MasterMind2k** [comment]
Related to #1.

File: .pylintrc [hunk 3]
```diff
@@ -0,0 +1,378 @@
...
+# Python code to execute, usually for sys.path manipulation such as
+# pygtk.require().
+#init-hook=
+
+# Add files or directories to the blacklist. They should be base names, not
+# paths.
+ignore=CVS
+
+# Pickle collected data for later comparisons.
+persistent=yes
```
Comment (MasterMind2k) [inline .pylintrc]: Put to no.

File: .pylintrc [hunk 4]
```diff
@@ -0,0 +1,378 @@
...
+# disable everything first and then reenable specific checks. For example, if
+# you want to run only the similarities checker, you can use "--disable=all
+# --enable=similarities". If you want to run only the classes checker, but have
+# no Warning level messages displayed, use"--disable=all --enable=classes
+# --disable=W"
+disable=import-star-module-level,old-octal-literal,oct-method,print-statement,unpacking-in-except,parameter-unpacking,backtick,old-raise-syntax,old-ne-operator,long-suffix,dict-view-method,dict-iter-method,metaclass-assignment,next-method-called,raising-string,indexing-exception,raw_input-builtin,long-builtin,file-builtin,execfile-builtin,coerce-builtin,cmp-builtin,buffer-builtin,basestring-builtin,apply-builtin,filter-builtin-not-iterating,using-cmp-argument,useless-suppression,range-builtin-not-iterating,suppressed-message,no-absolute-import,old-division,cmp-method,reload-builtin,zip-builtin-not-iterating,intern-builtin,unichr-builtin,reduce-builtin,standarderror-builtin,unicode-builtin,xrange-builtin,coerce-method,delslice-method,getslice-method,setslice-method,input-builtin,round-builtin,hex-method,nonzero-method,map-builtin-not-iterating
```
Comment (MasterMind2k) [inline .pylintrc]: Each warning/error in its own line.
```
disable=import-star-module-level\
        ,old-octal-literal\
...   
```

File: .pylintrc (hunk 3 above)
Comment (thrunabilax) [reply .pylintrc]: Done

File: .pylintrc (hunk 4 above)
Comment (thrunabilax) [reply .pylintrc]: Done

**Pull request merged**